CHANGELOG.md

```diff
@@ -2,6 +2,41 @@
 
 All notable changes to this project are documented in this file.
 
+## 1.0.0
+
+**Release date:** 2023-07-03
+
+This is the first stable release of the controller.
+
+It includes several minor changes that primarily focus on addressing forgotten
```

Comment:
```suggestion
This release includes several minor changes that primarily focus on addressing forgotten
```

Reply:
The release cadence will be here https://deploy-preview-1543--fluxcd.netlify.app/flux/releases/ Maybe we need to merge the website PR first.